tests/js/spec/components/events/eventTagsAndScreenshot.spec.tsx

```diff
@@ -268,6 +286,21 @@ describe('EventTagsAndScreenshot', function () {
         `/api/0/projects/${organization.slug}/${project.slug}/events/${event.id}/attachments/${attachments[1].id}/?download`
       );
 
+      // Display help text when hovering question element
+      userEvent.hover(screen.getByTestId('more-information'));
+
+      expect(
+        await screen.findByText(
+          'This image was captured around the time that the event occurred.'
+        )
+      ).toBeInTheDocument();
+
+      // Screenshot is clickable
+      userEvent.click(screen.getByRole('img'));
+
+      // Open modal
+      expect(screenshotModal).toHaveBeenCalled();
```

Comment:
Would this test closer resemble the way this feature is used if we asserted that some text from the modal is visible on screen instead of asserting that some internal function for opening the modal was called?

Reply:
yes, we could do that too. I can update it